scripts/sys_temp.sh

```diff
@@ -3,7 +3,13 @@
 export LC_ALL=en_US.UTF-8
 
 get_temp() {
-  echo "$(vcgencmd measure_temp | sed 's/temp=//')"
+  if [[ -d "/proc/device-tree/" && -f "/proc/device-tree/model" && $(grep -i 'Raspberry' /proc/device-tree/model) ]]; then
+    # It's a Raspberry pi
+    echo "$(vcgencmd measure_temp | sed 's/temp=//')"
+  else
+    # It's a Regular pc
```

Comment:
How about I have linux on a Mac? 😄 
I think it should be "It's anything else" or just remove the comment. Your choice.